File: launchdarkly/php/src/GetContextsExample.php
<?php

namespace OSEG\LaunchDarklyExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use LaunchDarkly;

$config = LaunchDarkly\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("ApiKey", "YOUR_API_KEY");

try {
    $response = (new LaunchDarkly\Client\Api\ContextsApi(config: $config))->getContexts(
        project_key: null,
        environment_key: null,
        kind: null,
        key: null,
        limit: null,
        continuation_token: null,
        sort: null,
        filter: null,
        include_total_count: null,
    );

    print_r($response);
} catch (LaunchDarkly\Client\ApiException $e) {
    echo "Exception when calling Contexts#getContexts: {$e->getMessage()}";
}

File: launchdarkly/python/src/GetContextsExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.ContextsApi(api_client).get_contexts(
            project_key=None,
            environment_key=None,
            kind=None,
            key=None,
            limit=None,
            continuation_token=None,
            sort=None,
            filter=None,
            include_total_count=None,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling ContextsApi#get_contexts: %s\n" % e)
